tests/base/pom.xml

```diff
@@ -124,6 +124,11 @@
             <artifactId>jboss-logmanager</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Would be better not to require this; and instead just use FasterXML as we use everywhere else